manta-trusted-setup/src/util.rs

```diff
@@ -49,8 +51,7 @@ pub trait HasDistribution {
 /// interface for building a serialization over the type `T`. For deserialization see the
 /// [`Deserializer`] `trait`.
 ///
-/// [`CanonicalSerialize`]: ark_serialize::CanonicalSerialize
-/// [`CanonicalDeserialize`]: ark_serialize::CanonicalDeserialize
+/// [`CanonicalDeserialize`]: manta_crypto::arkworks::serialize::CanonicalDeserialize
```

Comment:
Missed [`CanonicalSerialize`]?

Reply:
It's not needed here since it's imported.